portality/templates/publisher/uploadmetadata.html

```diff
@@ -104,6 +104,7 @@
                     <td>
                         {% if file.error %}
                             {{file.error}}
+                            <a href="{{ url_for('doaj.publishers', _anchor=file.error) }}">(explain)</a>
```

Comment:
Ahaaaa, that's why the anchors have full sentences with spaces for names ... very clever.